services/api-server/src/simcore_service_api_server/core/settings.py

```diff
@@ -137,6 +137,7 @@ class ApplicationSettings(BasicSettings):
     )
     API_SERVER_LOG_CHECK_TIMEOUT_SECONDS: NonNegativeInt = 3 * 60
     API_SERVER_PROMETHEUS_INSTRUMENTATION_ENABLED: bool = True
+    API_SERVER_PROMETHEUS_INSTRUMENTATION_COLLECT_SECONDS: PositiveInt = 5
```

Comment:
not sure where this is exactly used, could you please quickly comment?

Reply:
The metrics which are added here are not related directly to requests. So I cannot collect them using middleware. I could collect them directly in the `/metrics` endpoint of the api-server. But that endpoint is added by code inside `servicelib`. Hence, I need to generate/collect these metrics somewhere. I decided to create a task which runs in the background of the api-server and collects the metric. `API_SERVER_PROMETHEUS_INSTRUMENTATION_COLLECT_SECONDS` is the number of seconds the task should wait between collecting the metrics.

Reply:
So it is cached, and the cache is updates every 5 seconds? If so, very nice, feel free to resolve this convo and thanks for answering the questions

Reply:
yep